app/controllers/physical_storage_controller.rb

```diff
@@ -4,12 +4,35 @@ class PhysicalStorageController < ApplicationController
   include Mixins::GenericSessionMixin
   include Mixins::MoreShowActions
   include Mixins::BreadcrumbsMixin
+  include Mixins::GenericButtonMixin
 
   before_action :check_privileges
   before_action :session_data
   after_action :cleanup_action
   after_action :set_session_data
 
+  def specific_buttons(pressed)
+    case pressed
+    when 'physical_storage_new'
+      javascript_redirect(:action => 'new')
+    else
+      return false
+    end
+    true
+  end
+
+  def new
+    assert_privileges("physical_storage_new")
+    @physical_storage = self.class.model.new
+    session[:changed] = nil
+    @in_a_form = true
+    if params[:storage_manager_id]
+      @storage_manager = find_record_with_rbac(ExtManagementSystem, params[:storage_manager_id])
+    end
```

Comment:
Also unnecessary, probably needed only for `edit` in the future.

Reply:
removed "if params ..." but lived the "@in_a_form = true".
 when removing "@in_a_form = true" the UI displayed with an empty area on the left side